src/kOS/Suffixed/KUniverseValue.cs

```diff
@@ -46,12 +47,13 @@ public void InitializeSuffixes()
             AddSuffix("GETCRAFT", new TwoArgsSuffix<CraftTemplate, StringValue, StringValue>(GetCraft));
             AddSuffix("LAUNCHCRAFT", new OneArgsSuffix<CraftTemplate>(LaunchShip));
             AddSuffix("LAUNCHCRAFTFROM", new TwoArgsSuffix<CraftTemplate, StringValue>(LaunchShip));
+            AddSuffix("LAUNCHCRAFTWITHCREWFROM", new ThreeArgsSuffix<CraftTemplate, ListValue, StringValue>(LaunchShip));
```

Comment:
Can you write a short description of what user-facing documentation should look like for this?
Like what is expected when you pick crew names that don't exist.  What is expected when the number of crew in the argument doesn't match the number of seats in the vessel, how will crew get assigned (I looked at the code and it appears it prioritizes putting pilots in command seats, saving passenger seats for the other types, but I'm not sure.)